backend/src/main/java/com/google/sps/notifiers/Notifier.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main.java.com.google.sps.notifiers;
+
+public class Notifier {
+
+  private String name; /* String representing the user who is to be notified's name */
```

Comment:
Please make this javadocs by switching to `/** */` instead of `/* */` and move it above the field.

Reply:
Done